templates/patterns/list_item/pattern-list-item.html.twig

```diff
@@ -2,12 +2,17 @@
 /**
  * @file
  * Default item list component.
+ *
+ * @see @ecl/card
  */
 #}
-{% include '@ecl/ec-component-list-item' with {
-  'href': url,
-  'metas': meta,
-  'title': title,
-  'detail': detail,
-  'screen_reader': 'List item'|t
+{% include '@ecl/card' with {
+  card : {
+    description: detail,
+    title: {
+      label: title,
+      path: url
+    },
+    meta: meta
+  }
```

Comment:
This should also be done with content item composition, we use the card only on the highlight variant.

Reply:
ok